### PR TITLE
Add a bit more information

### PR DESCRIPTION
**Pull request by atoulme**

### CODE REVIEW

File: src/multiproof.rs [hunk 1]
```diff
@@ -37,6 +37,9 @@ impl CRS {
         CRS { n, G, Q }
     }
 
+    /*
+     Evaluates the polynomial at a specific point.
```
Comment (kevaundray) [inline src/multiproof.rs]: A commitment to a polynomial is different from evaluating that polynomial, so this comment would not be correct

Comment (atoulme) [reply src/multiproof.rs]: Sure thing. What would be the correct wording here?

Comment (kevaundray) [reply src/multiproof.rs]: Sorry about the late reply. Comment can say "Commits to a polynomial in lagrange form"

Comment (atoulme) [reply src/multiproof.rs]: OK, addressed.

**Review by kevaundray — APPROVED** [review]

**Pull request merged**